templates/includes/card-group-promo--green.html

```diff
@@ -0,0 +1,33 @@
+{% load wagtailimages_tags %}
+
+{% image page.teaser_image fill-446x273 as teaser_image_small %}
+{% image page.teaser_image fill-492x301 as teaser_image_medium %}
+{% image page.teaser_image fill-508x311 as teaser_image_large %}
+{% image page.teaser_image fill-626x383 as teaser_image_extra_large %}
+
+<li class="card-group-promo--green">
+    <h2 class="card-group-promo__heading">{{ heading }}</h2>
+
+    <div class="card-group-promo__card">
+        <div class="row">
+            <div class="col-md-12 col-lg-7 col-xl-6">
+                <a href="{{ page.url }}" tabindex="-1" class="card-group-promo__card-link">
+                    <picture class="mt-auto">
```

Comment:
Similar to the comment above, I think the margin for this `picture` element should be within the Sass for the component.

Reply:
Thanks @gtvj, looks like the class is not needed with the new styles anyway so I have removed it.